test_benchmarks.py

```
@@ -41,3 +45,88 @@
+            mock_get.return_value = mock_response
+
+            # Create scraper with cache enabled
+            scraper = MarkdownScraper(cache_enabled=True)
+            scraper.request_cache.cache_dir = Path(temp_dir)  # Override cache directory
+
+            # Trigger initial request to populate cache
+            url = "http://example.com/benchmark"
+            scraper.scrape_website(url)
+
+            # Benchmark subsequent requests
+            def scrape():
```

Ruff (0.8.2)

Missing return type annotation for private function `scrape`

(ANN202)